test/end-to-end/data-sources.test.ts

```diff
@@ -65,7 +65,6 @@ describe('DataSources', () => {
 
       localStack = await new LocalstackContainer('localstack/localstack:3')
         .withNetwork(network as any)
-        .withName('localstack')
```

Comment:
this was causing warnings/errors when attempting to spin up local stack. this will give it a dynamic name for each execution avoiding conflicts

```bash
Error: (HTTP code 409) unexpected - Conflict. The container name "/localstack" is already in use by container "420421cdc2b4abb71994d9034928ab0fd352c77cc1e0cc8fe05eff9112297135". You have to remove (or rename) that container to be able to reuse that name. 
```